java/com/google/location/lbs/activity/audioset/dataprocessing/incomplete-example.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+option java_outer_classname = "IncompleteExample";
+
+message Example {
+    string ytid = 1;
+    int32 start = 2;
```

Comment:
ditto. the field name should be more descriptive -- is it starting time of a clip? label? seconds? milliseconds? if it's hard to contain all the info, then add comments.

also if it's time, int64 is preferred.